pkg/occlient/occlient.go

```diff
@@ -678,26 +679,38 @@ func (c *Client) GetLatestBuildName(buildConfigName string) (string, error) {
 }
 
 // StartBinaryBuild starts new build and streams dir as source for build
-func (c *Client) StartBinaryBuild(name string, dir string) error {
+// asFile indicates if it is a binary build or not
```

Comment:
You are always starting binary build. Maybe explain that `asFile` indicates that   it will start build with single file specified in `path` parameter, otherwise it assumes that `path` is a directory